java/arcs/sdk/android/storage/ServiceStore.kt

```diff
@@ -149,11 +166,18 @@ class ServiceStore<Data : CrdtData, Op : CrdtOperation, ConsumerData>(
     override suspend fun onProxyMessage(message: ProxyMessage<Data, Op, ConsumerData>): Boolean {
         val service = checkNotNull(storageService)
         val result = DeferredResult(coroutineContext)
+        outgoingMessages.incrementAndGet()
         channel.send {
             service.sendProxyMessage(message.toProto().toByteArray(), result)
         }
         // Just return false if the message couldn't be applied.
-        return try { result.await() } catch (e: CrdtException) { false }
+        return try {
+            result.await()
+        } catch (e: CrdtException) {
+            false
```

Comment:
Should we be logging these? Could the silent swallow of these be behind any flakes?

Reply:
I don't think this is a smoking gun for some flakiness... but logging could be useful, even if it's just at debug level during tests.